go.mod

```diff
@@ -0,0 +1,13 @@
+module gopkg.in/src-d/enry.v1
```

Comment:
I would rather postpone this because we probably want to switch back to `github.com` imports. There is really no point in using `gopkg.in` with modules.

Reply:
Ok. I've just remove module files and left go-oniguruma instead of rubex